i18n/cs/docusaurus-plugin-content-docs/current/adguard-browser-extension/installation.md

```diff
@@ -21,7 +21,7 @@ V otevřeném dialogovém okně klikněte na _Přidat rozšíření_.
 
 ## Jak nainstalovat Rozšíření prohlížeče AdGuard pro Operu
 
-Otevřete stránku <a "https://agrd.io/extension_opera">addons.opera.com</a>.
+Otevřete stránku [addons.opera.com](https://agrd.io/extension_opera).
```

Comment:
please don't make changes to the files in the i18n folder
we can only change the English version, all the rest is fixed on Crowdin